src/analysis/special_functions/gamma/gamma.lean

```diff
@@ -0,0 +1,673 @@
+/-
```

Comment:
maybe name this file `basic.lean` instead of `gamma.lean`.

Reply:
Done